src/control/directive.spec.ts

```diff
@@ -263,6 +263,27 @@ describe(NgrxFormControlDirective.name, () => {
     });
   });
 
+  describe('ngrxUpdateOn "never"', () => {
+    beforeEach(() => {
+      directive.ngOnInit();
+      directive.ngrxFormControlState = { ...INITIAL_STATE, isTouched: true, isUntouched: false };
+      directive.ngrxUpdateOn = NGRX_UPDATE_ON_TYPE.NEVER;
+    });
+
+    it(`should not dispatch any even if the view value changed`, done => {
```

Comment:
The word `action` is missing here, i.e. `should not dispatch any action even if the view value changed`